quattor/types/component.pan

```diff
@@ -2,6 +2,8 @@ declaration template quattor/types/component;
 
 include 'quattor/functions/validation';
 
+type caf_service_action = string with match(SELF, '^(restart|reload|stop_sleep_start)$');
```

Comment:
use choice?

Reply:
Sure, as this is just a move I wasn't going to change the definition, but if you think it's worth it.

Reply:
this list is also not complete and best to choose otehr name to avoid conflict with metaconfig typ
use eg
```pan
type caf_service_actions = choice('start', 'stop', 'restart', 'reload', 'stop_sleep_start');
```

Reply:
Is it really a good idea to allow 'stop' or 'start' as an action to be run when something has changed though?